index.html

```diff
@@ -7,10 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:wght@200&display=swap" rel="stylesheet"> 
 </head>
-
 <body>
     <!-- Add your markup here -->
-</body>
-
+    <header class="header-container">
+        <div class="logo">
+            <img 
+            src="/img/mint-cupcake-logo2.png" 
```

Comment:
Having the same problem here of pictures not appearing until I change this to `img/mint-cupcake-logo2.png` 😩 just curious how this is working for you but not for me 🤔

Reply:
Ohh Sorry. It stays there when I "/" and call the link. I'll correct them.